thrift.js

```diff
@@ -202,6 +310,19 @@ Thrift.prototype.link = function link() {
     var self = this;
     var index;
 
+    if (self.linked) {
+        return self;
+    }
+    self.linked = true;
+
+    var moduleNames = Object.keys(self.modulesByName);
+    for (index = 0; index < moduleNames.length; index++) {
+        var thriftModule = self.modulesByName[moduleNames[index]];
+        thriftModule.link(self);
+        self.modulesByName[moduleNames[index]] = thriftModule;
+        self[moduleNames[index]] = thriftModule;
```

Comment:
Let’s just not add anything for now. We will probably have to retract everything but Services. Alternately, not expose anything at all.